app/weather_api.rb

```diff
@@ -4,24 +4,20 @@
 require 'pry'
 # require_relative './config/application'
 
-
 # class WeatherApi < Sinatra::Base
 
+get '/api/v1/weather/new_york' do
+  conn = Faraday.new(url: 'http://api.openweathermap.org')
 
-  get '/api/v1/weather/new_york' do
-    conn = Faraday.new(url: "http://api.openweathermap.org")
-
-    response = conn.get("/data/2.5/weather?") do |req|
-      req.params["q"] = "New York"
-      # req.params["appid"] =
+  response = conn.get('/data/2.5/weather?') do |req|
+    req.params['q'] = 'New York'
+    # req.params["appid"] =
 
-      req.params["appid"] = ENV['WEATHER_API_KEY']
-    end
+    req.params['appid'] = ENV['WEATHER_API_KEY']
   end
+end
```

Comment:
Curious to know if this code/file is necessary given that it looks like we call the API in `weather_service.rb` and set the latitude and longitude params there instead of city.